### ISSUE TITLE
Encourage feedback

### ISSUE THREAD

**Issue by filiprd** [dgarijo/VideoGameOntology#4]
Should we put something like "Feedback" subsection and encourage people to give comments and feedback?

**Comment by dgarijo** [comment]
Yes, maybe pointing to this github page, or giving an email. I think it is necessary.

**Comment by filiprd** [comment]
I would give both, saying that we look forward to any kind of feedback, and that it can be done by contacting us or on github page.

Also, maybe it would be useful to create a google group so that people can join and discuss, it is possibly the most effective way to do so. What do you think? In that case also people would not have to bother to come to github (and you need account to post here, right?), but just contact us and discuss.

**Comment by dgarijo** [comment]
For the moment I think the google group is not necessary. Nobody has shown interest in contributing yet.

**Comment by Japskua** [comment]
Yeah, I think we should consider creating discussion groups if this draws more attention.
